internal/storage/sql/common/flag.go

```diff
@@ -54,7 +59,7 @@ func (s *Store) GetFlag(ctx context.Context, key string) (*flipt.Flag, error) {
 
 	if err != nil {
 		if errors.Is(err, sql.ErrNoRows) {
-			return nil, errs.ErrNotFoundf("flag %q", key)
+			return nil, errs.ErrNotFoundf("flag %q; namespace %q", key, namespaceKey)
```

Comment:
Pedantic mode: How about switching the order? Just to preserve left-to-right precedence.
```suggestion
			return nil, errs.ErrNotFoundf("namespace %q; flag %q", namespaceKey, key)
```
or even something like the following:
```suggestion
			return nil, errs.ErrNotFoundf(`flag "%s/%s"`, namespaceKey, key)
```